pages/tag_selector.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Tag Selector
+authors:
+  - Mitchell Krome
+keywords: Tagging
+edited: 4 January 2017
+---
+
+The tag selector is opened from the view menu.
+It allows the selection of objects based on their tags.
+
+![ ](images/tag_selector.png)
+
+# Editor Layout
+
+Each row in the editor represents a new operation in the query.
+The columns define the required elements of the operation.
+
+## Relation
+
+The relation column specifies a logical operator describing how it is related to the row above.
+The first row therefore has no relation.
```

Comment:
Move "therefore" to the beginning of line? (Are you a native speaker? I am not.)